scripts/jobs/cron_tasks.inc.http.40.nginx_proxy.php

```diff
@@ -0,0 +1,1166 @@
+<?php
```

Comment:
There is lots of nginx config stuff done that is also done in the nginx-cron...maybe you can override just a view things instead of duplicating? Like we do with apache and apache_fcgid (class apache_fcgid extends apache). Could be possible